src/main/java/com/github/danielflower/mavenplugins/release/LocalGitRepo.java

```diff
@@ -20,13 +20,88 @@
 public class LocalGitRepo {
 
     public final Git git;
-    private final String remoteUrl;
     private boolean hasReverted = false; // A premature optimisation? In the normal case, file reverting occurs twice, which this bool prevents
-    private Collection<Ref> remoteTags;
+    private Collection<Ref> tags;
```

Comment:
Renamed `remoteTags` to just `tags` because they're not necessarily remote any more.